src/lib/plugins/links.ts

```diff
@@ -1,44 +1,79 @@
 import { readFileSync } from 'node:fs';
 import { join } from 'node:path';
+import type { Parent } from 'unist-util-visit/lib';
 
 import { DOCS_DIRECTORY } from '../../config/constants';
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function handleLinks(node: any, dirname: string) {
+export function handleLinks(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  node: any,
+  dirname: string,
+  idx: number | null,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  parent: Parent<any, any>
+) {
   let newUrl: string | null = null;
 
-  if (!node.url.includes('http')) {
-    newUrl = node.url
-      .replace('.md', '')
-      .replace('/index', '')
-      .replace('.html', '');
+  if (node.type === 'html') {
+    const url = getUrl(node.value);
+    if (url && idx) {
+      node.type = 'link';
+      node.url = url.includes(
+        'fuelbook.fuel.network/master/quickstart/developer-quickstart'
```

Comment:
Why we need this url here? Can we get from some config file?

Reply:
this is a temporary fix until we start using the latest version of the sway book where this is fixed

Reply:
We can add a TODO comment to help track this type of thing helping us to improve in the future